### ISSUE TITLE
Output log file

### ISSUE THREAD

**Issue by Jtachan** [MarcoGorelli/cython-lint#114]
Analogous to `flake8`, I wanted to output all the possible warnings in a .log file. However, I've seen that this is not possible yet.
My proposal is to add this feature with the same argument as in flake8:
```python
# Call on flake8
flake8 --output-file=flake8.log src

# Proposal for call on cython-lint
cython-lint --output-file=cylint.log src
```

I believe this is a nice feature to add, as other linters like pylint also allow to create log files by using `--output=pylint.log`.

**Comment by MarcoGorelli** [comment]
hi - does it not work to just pipe to a file, like `cython-lint src > cylint.log`?

**Comment by Jtachan** [comment]
Yes, that works. Thanks!
So I personally like the `--output` argument, but knowing this use case I understand if this feature won't be implemented.

**Comment by MarcoGorelli** [comment]
cool, thanks - closing then, but thanks fr the request
